packages/angular_devkit/build_webpack/src/webpack-dev-server/index_spec.ts

```diff
@@ -41,7 +39,9 @@ describe('Dev Server Builder', () => {
   }
 
   beforeEach(async () => {
-    await createArchitect(path.join(devkitRoot, 'tests/angular_devkit/build_webpack/basic-app/'));
+    const ngJsonPath = path.join(path.dirname(__filename), '../../test/basic-app/angular.json');
```

Comment:
Nit: `path.dirname(__filename)` -> `__dirname`

Reply:
Fixed. Think I left that in because at some point I was resolving the data files on windows, but these tests don't run on windows right now so that's just noise.